tests/end2end/features/test_caret_bdd.py

```diff
@@ -17,15 +17,10 @@
 # You should have received a copy of the GNU General Public License
 # along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
 
+# pylint: disable=unused-import
```

Comment:
Why not simply remove the `pytest` import now that it's not needed anymore? :wink:

Reply:
Yeah, forgot about this.